voltdbclient/url_test.go

```diff
@@ -23,28 +23,33 @@ func TestParseURL(t *testing.T) {
 	}
 
 	for _, s := range sample {
-		u, err := parseURL(s.conn)
-		if s.pass {
-			if err != nil {
-				t.Fatal(err)
+		s := s //capture variable
+		t.Run("", func(t *testing.T) {
```

Comment:
I agree with you. But don't you think we are  going extreme here. I mean I can voice how I  use  `t.Run` and `t.Parallel`

-  named test case: most of the cases I fall back to `t.Run` when I can name that test case that way I can visually investigate the test output output. For instance `t.Run("some boring test", ...`

- wrapping `setup` and `teardown` . here is a snippet I took from one of the tests on my personal project 
   ```go
   db, err := d.Open()
	if err != nil {
		t.Fatal(err)
	}
	t.Run(db.Dialect().GetName(), func(ts *testing.T) {
		f(ts, db)
	})
	err = d.Clear(tables...)
	if err != nil {
		t.Fatal(err)
	}
   ```
  As you can see, I setup the database and tear it down when done. The project is an ORM and I use this style to test multiple databases( postgresql,ql etc)  [source](https://github.com/ngorm/ngorm)

- tests that take more than 1 second: I use this criterial to set `t.Parallel`

I suggest we don't use `t.Run` we can still use `t.Parallel`  I will comment where I believe it will be more useful since this test doesn't depend on any global state or other tests,.

Something like

```go
func TestParseURL(t *testing.T) {
	t.Parallel()
...
```